src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/media-edit-overlay/cropping-slide.js

```diff
@@ -0,0 +1,516 @@
+/*
+ * This file is part of the Sulu CMF.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+/**
+ * @class CroppingSlide
+ * Class which handles the cropping-slide in the media-edit overlay.
+ * Note that this is not an aura-component but a simple require-js dependency.
```

Comment:
But why isn't it an own aura component then? Asking because the essentials of a component (`sandbox` and `$el`) are already passed anyway.